hackIDE/static/hackIDE/js/custom.js

```diff
@@ -269,7 +269,13 @@ $(document).ready(function(){
 						$(".output-io-info").hide();
 						$(".compile-status").children(".value").html("--");
 						$(".error-key").html("Compile error");
-						$(".error-message").html(response.compile_status);
+						
+						var compileMsgResponse = response.compile_status;
+						if (response.compile_status == "" || response.compile_status.length <= 1) {
+							compileMsgResponse = "An error ocurred while compiling your code or your code is not correct";
```

Comment:
We can make this explicit for the end user when we receive empty compile response. Wouldn't that be a better scenario? If so, we can change it to something like `"Empty Compile Response. Something went wrong while compiling"`.